### ISSUE TITLE
Failure to parse long float

### ISSUE THREAD

**Issue by rekka** [uiri/toml#55]
Hi!

I'm using version 0.9.1 from pypi. The following fails:

```python
>>> import toml
>>> toml.loads('x=0.1234567891234567891')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bertik/.local/lib/python2.7/site-packages/toml.py", line 331, in loads
    value, vtype = load_value(pair[1])
  File "/home/bertik/.local/lib/python2.7/site-packages/toml.py", line 453, in load_value
    parsed_date = load_date(v)
  File "/home/bertik/.local/lib/python2.7/site-packages/toml.py", line 350, in load_date
    tz = TomlTz(val[19:24])
  File "/home/bertik/.local/lib/python2.7/site-packages/toml.py", line 10, in __new__
    self._minutes = int(toml_offset[4:6])
ValueError: invalid literal for int() with base 10: ''
```

It seems that it is trying to parse a long float as a date.

Shorter floats of course work:

```python
>>> toml.loads('x=0.123456789123456789')
{'x': 0.12345678912345678}
```

**Comment by uiri** [comment]
This issue appears to be fixed in the latest version. Can you confirm whether or not it is present in release 0.9.2?

**Comment by rekka** [comment]
Seems to be fixed in 0.9.2:

```
import toml
toml.loads('x=0.1234567891234567891')
{u'x': 0.12345678912345678}
```

**Comment by uiri** [comment]
Thanks for confirming!
Closed as FIXED.
